educator/src/Dscp/Educator/Logic/Publishing.hs

```diff
@@ -46,7 +46,7 @@ publishMissingBlocks =
         let pubTxs = createPublicationTxw feePolicy sk <$> blocks
 
         logInfo . fmt $ nameF "Adding publications to mempool" (listF $ toPtxId . ptwTx <$> pubTxs)
-        forM_ pubTxs $ addTxToMempool . GPublicationTxWitnessed
+        forM_ pubTxs $ relayTx . GPublicationTxWitnessed
```

Comment:
Actually, I'm still wrong here. It should be 
```
forM_ pubTxs $ \pubTx -> relayTx (GPublicationTxWitnessed pubTx) >>= wait @"tx in mempool"
```

Otherwise, since `updateMempoolWithPublications` is invoked every second, on high load when transactions which are put to the relay does not appear in mempool immediately, we will effectively fill the relay with many duplicates of our publications.

@pasqu4le do you mind if I make this change and force-push?

Reply:
@Martoon-00 no problem, please do.
Just please check that everything stay signed, because for some reason I seem to loose some signatures when rebasing or cherry-picking.
You could also add another commit on top of this, perhaps

Reply:
@Martoon-00 why didn't you proceed with this change?